pentestportal_sg_rules.tf

```
@@ -39,6 +39,21 @@ resource "aws_security_group_rule" "pentestportal_ingress_from_kali_via_https" {
   to_port           = 443
 }
 
+# Allow ingress from Kali instances via port 8080
+# For: Assessment team "development mode" access to pentest portal from
+# Kali instances
+resource "aws_security_group_rule" "pentestportal_ingress_from_kali_via_8080" {
+  count    = lookup(var.operations_instance_counts, "pentestportal", 0) > 0 ? 1 : 0
+  provider = aws.provisionassessment
+
+  security_group_id = aws_security_group.pentestportal.id
+  type              = "ingress"
+  protocol          = "tcp"
+  cidr_blocks       = formatlist("%s/32", aws_instance.kali[*].private_ip)
```

I think something like this would be more modern:
```suggestion
  cidr_blocks       = [for instance in aws_instance.kali : format("%s/32", instance.private_ip)]
```

My viewpoint is colored by the fact that I detest the splat notation, so feel free to ignore.

---

Thanks for the excellent suggestion - changed in https://github.com/cisagov/cool-assessment-terraform/pull/76/commits/c698d76fd6a6ae0d4e4aceda9b10db31cd2c8df5.